src/js/background.js

```diff
@@ -766,6 +789,51 @@ Badger.prototype = {
     chrome.browserAction.setTitle({tabId: tab.id, title: "Privacy Badger"});
   },
 
+  showCookies: function(fqdn) {
+    let domain = window.getBaseDomain(fqdn);
+    chrome.cookies.getAll({domain}, cookies => {
+      console.log('Cookies for domain: ' + domain);
+      cookies.forEach(cookie => {
+        console.log('\t -------------');
+        console.log('\t cookie name: ' + cookie.name);
+        console.log('\t cookie domain: ' + cookie.domain);
+      });
+    });
+  },
+
+  debugSite: function() {
+    let self = this;
+    chrome.tabs.query({'active': true}, tabs => {
+      tabs.forEach(tab => {
```

Comment:
This could help us test our debugging function by separating `console.log` calls from data retrieval. Would also rescue another indentation level by making `_debug` above a public-facing function.

We should add a couple of basic tests for expected output, just so we don't overlook our debugging code after making changes to `action_map`/`snitch_map` structures.